uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -86,9 +87,16 @@ fun FeedScreen(
     effectFlow.collectInLaunchedEffect { effect ->
         when (effect) {
             is FeedViewModel.Effect.ErrorMessage -> {
-                scaffoldState.snackbarHostState.showSnackbar(
-                    effect.appError.getReadableMessage(context)
-                )
+                when (
+                    scaffoldState.snackbarHostState.showSnackbar(
+                        message = effect.appError.getReadableMessage(context),
+                        actionLabel = "Reload",
+                    )
+                ) {
+                    SnackbarResult.ActionPerformed -> dispatch(FeedViewModel.Event.ReloadContent)
```

Comment:
Almost LGTM! 👍 
Can you also add a snackbar that says "not yet implemented" here? This is because the user thinks that it is a bug that cannot be fixed even if it is reloaded, causing confusion. 🙇

Reply:
OK!